Winleafs.Api/DTOs/Shared/WriteObjectDto.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Winleafs.Api.DTOs.Shared
+{
+	public class WriteObjectDto
```

Comment:
Can this be more specific or is it used for all writes? Then maybe make this a base class and make one or more derived classes, which then has, for example, the `ExtControlVersion` property